.env.example

```
@@ -180,6 +189,19 @@ RAGAPP_PASSWORD=
  #####
    #
 
+############
+# LibreTranslate Configuration (optional)
+############
+LIBRETRANSLATE_CHAR_LIMIT=10000
+LIBRETRANSLATE_DEFAULT_SOURCE=auto
+LIBRETRANSLATE_DEFAULT_TARGET=en
+LIBRETRANSLATE_API_KEYS=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LIBRETRANSLATE_API_KEYS key should go before the LIBRETRANSLATE_CHAR_LIMIT key

(UnorderedKey)